utils/src/main/kotlin/Utils.kt

```diff
@@ -288,6 +268,14 @@ fun normalizeVcsUrl(vcsUrl: String): String {
     return url
 }
 
+/**
+ * Resolve the Windows [executable] to its full name including the optional extension.
+ */
+fun resolveWindowsExecutable(executable: File): File? {
```

Comment:
Should this function have an `if (Os.isWindows)` guard?

Reply:
I thought about that, but as the function name already contains "Windows" I believe it's clear to be Windows-specific, and I assume it to be used in a context where the `if (Os.isWindows)` check happened already at the caller side.

Finally, on non-Windows the function will anyway just return `executable` (if it's an existing file), so even without that guard I assume no real harm to be done.